forms-flow-components/src/components/CustomComponents/FormBuilderModal.tsx

```diff
@@ -76,8 +75,11 @@ export const FormBuilderModal: React.FC<BuildFormModalProps> = React.memo(
       display: checked ? "wizard" : "form"
     })
     const [cachedTitle, setCachedTitle] = useState("");
+    const [isLoading, setIsLoading] = useState(false);
+    const [isValidating, setIsValidating] = useState(false); 
```

Comment:
handle this from opensource : 

pass this as props : 
isSaveBtnLoading={formSubmitted}
isFormNameValidating = {validationLoading}

Reply:
Resolved